_includes/category.html

```diff
@@ -1,9 +1,9 @@
 {% assign category = site.data.categories[include.categoryText] %}
 {% assign displayPosts = site.posts | where: 'category', include.categoryText  %}
-{{category}}
 <div class="list-container">
     <div class="header">
       <div class="featured-container container">
+        {% include filters.html selected= 'mobile'%}
```

Comment:
Is the `selected` needed or is just based on the path?
```suggestion
        {% include filters.html%}
```